src/error.vue

```diff
@@ -1,109 +1,98 @@
-    <Meta
-      property="og:image:width"
-      content="1200"/>
-    <Meta
-      property="og:image:height"
-      content="600"/>
-    <Meta
-      property="og:image:alt"
-      :content="APP_TITLE"/>
-    <Meta
-      property="og:url"
-      :content="APP_URL"/>
-    <Meta
-      name="twitter:card"
-      content="summary_large_image"/>
-    <Meta
-      name="twitter:site"
-      :content="APP_CREATOR"/>
-    <Meta
-      name="twitter:creator"
-      :content="APP_CREATOR"/>
-  </Head>
-
-  <the-header/>
-  <div class="error">
-    <div class="error__parallax">
-      <page-header>
-        Unexpected Error
-      </page-header>
-
-      <app-panel class="error__panel">
-        <h2 class="error__heading">
-          Sorry! We encountered an unexpected error
-        </h2>
-        <p class="error__paragraph">
-          ERROR CODE: 500 - Internal Server error
-        </p>
-        <p class="error__paragraph">
-          We’re not quite sure what went wrong. You can go back, ask for help
-          in the
-          <app-link to="https://forum.aeternity.com/">
-            Forum
-          </app-link>
-          or tell us your story in the Feedback Survey.
-        </p>
-
-        <div class="error__container">
-          <app-button
-            to="/"
-            class="error__button">
-            Back to Dashboard
-          </app-button>
+  <div>
+    <Head>
+      <Title> {{ APP_TITLE_SHORT }} | Unexpected Error </Title>
+
+      <Meta
+        name="description"
+        :content="APP_DESCRIPTION"/>
+      <Meta
+        name="keywords"
+        :content="APP_KEYWORDS"/>
+      <Link
+        rel="apple-touch-icon"
+        sizes="180x180"
+        href="/favicon-apple-180.png"/>
+      <Link
+        rel="icon"
+        type="image/png"
+        sizes="512x512"
+        href="/favicon-512.png"/>
+      <Link
+        rel="icon"
+        type="image/png"
+        sizes="96x96"
+        href="/favicon-96.png"/>
+      <Link
+        rel="icon"
+        type="image/png"
+        sizes="32x32"
+        href="/favicon-32.png"/>
+      <Meta
+        property="og:type"
+        content="website"/>
+      <Meta
+        property="og:title"
+        :content="APP_TITLE"/>
+      <Meta
+        property="og:description"
+        :content="APP_DESCRIPTION"/>
+      <Meta
+        property="og:image"
+        :content="`${APP_URL}/social-cover.png`"/>
+      <Meta
+        property="og:image:width"
+        content="1200"/>
+      <Meta
+        property="og:image:height"
+        content="600"/>
+      <Meta
+        property="og:image:alt"
+        :content="APP_TITLE"/>
+      <Meta
+        property="og:url"
+        :content="APP_URL"/>
+      <Meta
+        name="twitter:card"
+        content="summary_large_image"/>
+      <Meta
+        name="twitter:site"
+        :content="APP_CREATOR"/>
+      <Meta
+        name="twitter:creator"
+        :content="APP_CREATOR"/>
+    </Head>
+
+    <NuxtErrorBoundary>
```

Comment:
Why this can't be simplified, you may ask. Great question. It's because on ALL of our error pages, we have still a fully interactive application with links, searches, and data from APIs which may again trigger some errors. If we simplified Error 500 to the bare minimum, all the other pages could be implemented through a single NuxtErrorBoundary on the app level, and all of that extra innerError logic would be removed.

Fortunately, the only "ugly" code is in the error.vue. Everything else looks clean to me.